.env.example

```diff
@@ -1,4 +1,7 @@
 production=false
+isDevelop=true
+isIntegration=false
+isPreProd=false
```

Comment:
just a thought - could we have a single variable to encapsulate this, something like environment: where the value is set to 'develop' 'integration' or 'preprod' instead of 4 separate properties?

Reply:
ah my bad! didn't have to add this, those values are calculated based on the environment in setenv.ts